src/pages/resume-page.md

```diff
@@ -0,0 +1,146 @@
+  - content: Udemy Ultimate Full Stack Developer Bootcamp
+  - content: StackSkills Developer Certifications
+  - content: Codecademy Web Development Certifications
+
+secondaryEducation: 
+  - content: 'Post-Bachelor Studies: Teaching Certification, Music Education'
+    classes: cmp-resume-page__sub-list-item--title
+  - content: Wayne State University
+    classes: util-margin-none
+  - content: Detroit, <abbr title="Michigan">MI</abbr>
+    classes: util-margin-none
+
+tertiaryEducation: 
+  - content: Bachelor of Music in Performance
+    classes: cmp-resume-page__sub-list-item--title
+  - content: Wayne State University
+    classes: util-margin-none
+  - content: Detroit, <abbr title="Michigan">MI</abbr>
+    classes: util-margin-none
+
+tech:
+  - image: '../public/usedresources/html.svg'
+    alt: 'HTML 5'
+    name: HTML
+  - image: '../public/usedresources/css.svg'
+    alt: 'CSS 3'
+    name: CSS
+  - image: '../public/usedresources/sass.svg'
+    alt: 'Sass/SCSS'
+    name: SCSS 
+  - image: '../public/usedresources/bootstrap.svg'
+    alt: 'Bootstrap 5'
+    name: Bootstrap 
+  - image: '../public/usedresources/javascript.svg'
+    alt: 'JavaScript'
+    name: JavaScript
+  - image: '../public/usedresources/typescript.svg'
+    alt: 'TypeScript'
+    name: TypeScript
+  - image: '../public/usedresources/react.svg'
+    alt: 'React'
+    name: React
+  - image: '../public/usedresources/nextjs.svg'
+    alt: 'Next.js'
+    name: Next.js
+  - image: '../public/usedresources/angular.svg'
+    alt: 'Angular'
+    name: Angular
+  - image: '../public/usedresources/vue.svg'
+    alt: 'Vue.js'
+    name: Vue.js
+  - image: '../public/usedresources/github-black.svg'
+    alt: 'Github'
+    name: Github
+  - image: '../public/usedresources/git.svg'
+    alt: 'Git'
+    name: Git
+  - image: '../public/usedresources/azure.svg'
+    alt: 'Azure Dev Ops'
+    name: Azure Dev Ops
+  - image: '../public/usedresources/nunjucks.svg'
+    alt: 'Nunjucks'
+    name: Nunjucks
+  - image: '../public/usedresources/handlebars.svg'
+    alt: 'Handlebars'
+    name: Handlebars
+  - image: '../public/usedresources/jest.svg'
+    alt: 'Jest'
+    name: Jest
+  - image: '../public/usedresources/cypress.svg'
+    alt: 'Cypress'
+    name: Cypress
+  - image: '../public/usedresources/figma.svg'
+    alt: 'Figma'
+    name: Figma
+
+currentExperienceHighLevels:
+  - classes: cmp-resume-page__list-item--heading
+    content: |
+      January 2023 &hyphen; Present
+  - classes: cmp-resume-page__list-item--title util-bold
+    content: Developer
+  - classes: util-margin-none
+    content: Sparkbox
+  - classes: util-margin-none
+    content: |
+      Bellbrook, <abbr title="Ohio">OH</abbr>
+currentExperienceDetails:
+  - content: 'Learning cohorts, Accessibility Guild, Foundry articles (Contrast), Lightning Talk presentation (React Design Patterns), internal demos (Figma recreations), internal tooling, client demos'
```

Comment:
This feels like "work in progress" text. Maybe add links for items with the parentheses?